CIP-015x/README.md

```diff
@@ -124,11 +124,11 @@ For the additional blocks (all except `HDR, CHUNK, MANIFEST`) it's possible to k
 
 - chunk size \~8–16MiB; footer required;
 - data is stored in deterministically defined global order; In the lexical order of the keys
-- all keys in the block must be unique
-- all key-values in the block must refer to the same namespace
+- all keys in the record must be unique
+- all key-values in the record must refer to the same namespace
 - readers should verify footer before relying on the data.
 - `chunk_hash = H(concat [ digest(e) | e in entries ])`
-- all CHUNKS blocks are ordered in lexicographical namespace order, to preserve canonical order
+- all values in `CHUNKS` record are ordered in lexicographical namespace order, to preserve canonical order
```

Comment:
Is this precisely the same thing? I guess what we're strictly saying is that all keys in CHUNK `n` must be lexicographically lower than all keys in CHUNK `n+1`?